src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/workflow/WorkflowScheduler.kt

```diff
@@ -9,25 +9,25 @@ class WorkflowScheduler(val workflow: Workflow) {
     private val instance = workflow.instance
     private val logger = workflow.logger
 
-    fun execute(nodes: Sequence<Node>): Int {
+    fun schedule(nodes: Sequence<Node>): Int {
```

Comment:
`Iterable<Node>` may be more compatible

Reply:
but Sequence is not chosen accidentally, it has lazy loading properties for nodes and generally is better for iterating over heavy elements